.changeset/add-import-fresh.md

```diff
@@ -0,0 +1,11 @@
+---
+"astro-integration-kit": minor
+---
+
+Adds `importFresh()` utility for cache-busting module imports during development.
+
+- **`importFresh()`**: Bypasses Node.js module cache by adding timestamp query parameters, essential for hot reloading without process restarts
+  ```ts
+  import { importFresh } from "astro-integration-kit/dev";
+  const { default: myIntegration } = await importFresh<typeof import("my-integration")>("my-integration");
+  ```
```

Comment:
Add a second bullet point explaining that it automatically tries with the extensions and `/index.*` suffixes for relative paths